docs/gallery_code/general/plot_anomaly_log_colouring.py

```diff
@@ -65,7 +62,7 @@ def main():
 
     # Use a standard colour map which varies blue-white-red.
     # For suitable options, see the 'Diverging colormaps' section in:
-    # http://matplotlib.org/examples/color/colormaps_reference.html
+    # http://matplotlib.org/stable/gallery/color/colormaps_reference.html
```

Comment:
This links does not exist anymore, I assume it has changed since `v2.0.2`.  The new link perhaps needs to be https://matplotlib.org/stable/gallery/color/colormap_reference.html#sphx-glr-gallery-color-colormap-reference-py

Reply:
@tkknight Nice one :eagle: :eyes: